### ISSUE TITLE
Sharing sidebar needs more space or separation

### ISSUE THREAD

**Issue by ChrisEdS** [owncloud/web#7110]
### Steps to reproduce
1. Share a document

### Expected behaviour
The sharing UI is nice and clean, you can see at a glance where an element ends and a new one begins.

### Actual behaviour
<img width="291" alt="image" src="https://user-images.githubusercontent.com/33026403/173063069-4eb77145-be63-4bc6-93cc-0a4065e673f5.png">

The view looks very pressed, the username almost protrudes into the permission schwitcher. Maybe it makes sense to create a visual separation with background contrast? But in any case there should be more space between the first two elements.

### Version
ownCloud Web 5.5 RC9

**Comment by kulmann** [comment]
cc @tbsbdr

**Comment by ChrisEdS** [comment]
This can also become a problem in different languages? So `Betrachter` can become longer in some cases. Moving this part in a new row might make sense.

**Comment by kulmann** [comment]
At some point the right sidebar had more space. But maybe it would also help to align the role switcher to the bottom of the row and the three dots button above it 🤔

**Comment by ChrisEdS** [comment]
Yeah, that sounds like a good idea! This is what I thought as well.

**Comment by kulmann** [comment]
In case of public links we have all configuration options in a line at the bottom and the link itself at the top. Maybe we need to give the sharee name the full row width at the top and put sharee type and options in the line below.

<img width="424" alt="Screenshot 2022-06-10 at 14 31 47" src="https://user-images.githubusercontent.com/3532843/173064706-53467c8d-214c-406c-bcf5-90a8f31a8dd8.png">

**Comment by ChrisEdS** [comment]
Why we have `User` or `Benutzer` under der sharee name? Which other variation or types of users are possible?

**Comment by ChrisEdS** [comment]
I see. That is needed then. Thanks for clarification.

**Comment by tbsbdr** [comment]
This is the current state of concept for a worstcase / very crowded scenario:

<img width="873" alt="Figma_2022-06-10_at_15-38-06@2x" src="https://user-images.githubusercontent.com/26610733/173077955-6d730951-a163-4444-b89c-f31b148441fe.png">

imo the ui above could be tidied up a lot if we decide to kill our darling, the 3-Dots and have everything within the dropdown:

<img width="746" alt="Whimsical_2022-06-10_at_15-42-19@2x" src="https://user-images.githubusercontent.com/26610733/173078692-28ad3f43-b20f-42fc-ac32-3b02ba9adbb1.png">

**Comment by tbsbdr** [comment]
**Proposal:**
- Without 3-Dots
- Icons more decent (not a11y compliant yet!)
- removed password-indicators from sharees (don't exist)
- moved indicators to the left
- actionable areas where primarily moved to the right
(still not perfect - draft status..)

**--> would you consider killing the 3-Dots an improvement?**

<img width="842" alt="Figma_2022-06-10_at_15-57-13@2x" src="https://user-images.githubusercontent.com/26610733/173082246-16063a8e-8976-451e-9f2f-71f9a1ee1f37.png">

**Comment by ChrisEdS** [comment]
Great stuff!

**Comment by kulmann** [comment]
@tbsbdr I don't get it, your mockup has huge amounts of horizontal space, while @ChrisEdS screenshot is a scenario with very little horizontal space.

**Comment by tbsbdr** [comment]
Jep, this is because the figma design was setup at 1920x1080 **hardware-Pixels** whereas Chris' (and many other laptop users resolution) might be ~1440 x 900 **CSS-Pixels** (equals 75% of 1920 width)

- **1440** should also be our target size for standard users (still not scientific-mode)

Scale down your browser zoom to 75% and you'll get the figma intended dimensions (which is to small imo)

If we tackle the dimension / scaling issue, I would recommend something like 85% compared to the current scale (~ 13-14 px for 1 REM)

**others:**
Onedrive, box and dropbox have 14px for standard text
Google drive has 13 px for standard text.
here in gh standard text is 14 px.

**Comment by JanAckermann** [comment]
Current POC:

![image](https://user-images.githubusercontent.com/26169327/178251157-00611aaa-89bb-456a-b460-072e04119725.png)

![image](https://user-images.githubusercontent.com/26169327/178251269-c7080451-867f-48af-8fbf-1477fb32978b.png)

**Comment by tbsbdr** [comment]
as discussed with @JanAckermann :
- we only show the most important information in the sharee list
- Move "Type" and "Additional Info" to the backseat (-> Access details)
- provide Info-i that shows on-click the "Access details"
- Access details can be extended in the future with more info like "Shared on" or "Shared by".
![image](https://user-images.githubusercontent.com/26610733/178471443-a5ef3632-3645-49f3-82aa-6bca83fb6c6c.png)
